lib/std/bool.lp

```diff
@@ -0,0 +1,57 @@
+// Data type of booleans.
```

Comment:
Could you please rename std into some thing else, say example? I don't want users to think that it could be the beginning of a standard library (even though it could be integrated in such a library later). Thank you.

Reply:
Well, the whole point was to have the stem of a minimal standard library. And I was hoping that someone would pick up on that to start building a collection of generally useful modules. I definitely makes sense to share very common stuff among all libraries, so why not start there?

Anyway, `std` is clearly documented as a very minimal standard library, I don't see how people could mistake that for anything else. And in all cases, I think that such a "standard library" will have to evolve with the tool. It is likely that developing it will make us realize that certain features are missing or not working well enough.

In any case, we can always decide to extend, rename or even remove `std` later. I dont think it is a problem to keep it as it is. Is it really a problem for you?

Reply:
Yes, it is.